cpu/esp32/include/cpu_conf.h

```diff
@@ -38,8 +38,12 @@ extern "C" {
  * @{
  */
 #define THREAD_EXTRA_STACKSIZE_PRINTF (1024)
+#ifndef THREAD_STACKSIZE_DEFAULT
```

Comment:
@kaspar30 It was intentionally not allowed to override these ESP32 settings because the values defined there are the minimum settings needed to work. If a user reduces them, he will run into problems without having an indication that the stack size is too small. What I could imagine is to use the minimum of a user defined size and the minimum required size here.

Reply:
I see. If I add a `#warning`, would that be fine? But actually I think it should be understood that lowering the default might cause issues.

Reply:
I agree. My regular case is increasing the sizes.

Reply:
> But actually I think it should be understood that lowering the default might cause issues.

Allright.